index.html

```diff
@@ -1,6 +1,11 @@
-<html>
-	<head>
-	</head>
-	<body>
-		<button> Try me </button>
-	</body>
\ No newline at end of file
+<!DOCTYPE html>
+	<html lang="en">
+		<head>
+   			 <meta charset="UTF-8">
+  			 <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  			  <title>Document</title>

```

Comment:
Ici, il faut mettre le nom du document, donc le nom du mini-jeu par exemple

Reply:
(je parlais de la ligne 6)
Du coup ça peut donner quelque chose comme `<title>Try to click</title>`